documentation/ntfs/ntfs_fr.md

```diff
@@ -465,6 +472,42 @@ admin_name | chaine | Requis | Nom de la commune (ou quartier).
 stop_id | chaine | Requis | Identifiant de la zone d'arrêt utilisée comme accroche de la commune (lien vers le fichier stops). Stop de type 1 oligatoirement.
 stop_name | chaine | Optionnel | Nom de la zone d'arrêt (pour faciliter la lisibilité du fichier)
 
+### pathways.txt (optionnel)
+Attention, ce fichier décrit une modélisation de la station qui n'est pas nécessairement géographique, et peut également être simplifiée.
```

Comment:
Minor: you can add a double space `  ` for layout.

Reply:
seems I can't do it with VS Code :p